File: Test_dir/readData.py
import csv, json

PATH_TO_CSV = "../Resources/Отзывы/AppStore (за год).csv"

def main():
    with open("../Resources/data.json", "r") as fd:
        data = json.load(fd)
    for d in data:
        d['readed'] = 0
    with open("../Resources/data.json", "w") as fd:
        fd.write(json.dumps(data))
    # i = 0
    # # data = {}
    # with open(PATH_TO_CSV, newline='', encoding='utf-8') as fd:
    #     reader = csv.reader(fd, delimiter=",")
    #     for row in reader:
    #         # data
    #         i += 1            
    #         if i > 5:
    #             break
    #         if i < 3:
    #             continue
    #         print(row)

if __name__ == "__main__":
    main()

File: Test_dir/readData.php
<?php
    $PATH_TO_CSV = "../Resources/Отзывы/AppStore (за год).csv";
    $PATH_TO_RES = "../Resources/data.json";
    $i = 0;
    $res = array();
    if (($handle = fopen($PATH_TO_CSV, "r")) !== FALSE) {
        while (($data = fgetcsv($handle, 1000, ",")) !== FALSE) {
            // $num = count($data);
            // print_r($data);
            // if ($i == 2)
            $new_data = array();
            $i++;
            // print_r($data);
            if ($i < 4)
                continue;
            // $res[]
            // foreach ($data as $col) {
                $new_data['Date'] = $data[0];
                // $new_data[''] = $data[1];
                // $new_data['Date'] = $data[2];
                $new_data['Country'] = $data[3];
                $new_data['Version'] = $data[4];
                $new_data['Author'] = $data[5];
                $new_data['Rating'] = $data[6];
                $new_data['Title'] = $data[7];
                $new_data['Review'] = $data[8];
                // $new_data['Tran'] = $data[9];
                // $new_data['Date'] = $data[10];
                $new_data['Reply Date'] = $data[11];
                $new_data['Developer Reply'] = $data[12];
                $new_data['User'] = $data[13];
                $new_data['Tags'] = $data[14];
                $new_data['Categories'] = $data[15];
                $new_data['Notes'] = $data[16];
                $new_data['Likes'] = $data[17];
                $new_data['Dislikes'] = $data[18];
                // $new_data['Reply Date'] = $data[20];
                // $new_data['Reply Date'] = $data[21];
            // }

            $res[] = $new_data;
            // if ($i == 6)
            //     break;
        }
        file_put_contents($PATH_TO_RES, json_encode($res));
    }
?>
